torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3338,11 +3380,8 @@ def method_tests():
         ('index_add', (), (0, torch.tensor(0, dtype=torch.int64), ()), 'scalar_all_dim', (), [0]),
         ('index_add', (S, S), (0, index_variable(2, S), (2, S)), 'alert_nondeterministic', (), [0],
             [expectedAlertNondeterministic('index_add_cuda_', 'cuda')]),
-        ('index_copy', (S, S), (0, index_perm_variable(2, S), (2, S)), 'dim', (), [0]),
         ('index_copy', (S, S), (0, index_perm_variable(2, S), (2, S)), 'dim_alert_nondeterministic', (), [0],
```

Comment:
I think it's OK to add this case without the expectedAlertNondeterministic as long as we file an issue that we want to create an OpInfo-based mechanism for testing nondeterminism and link that issue in a comment above the index copy sample inputs function.